djautotask/models.py

```diff
@@ -372,6 +375,32 @@ class TaskNote(TimeStampedModel, Note):
     )
 
 
+class Contact(TimeStampedModel):
+
+    first_name = models.TextField(blank=True, null=True, max_length=20)
+    last_name = models.TextField(blank=True, null=True, max_length=20)
+    middle_initial = models.TextField(blank=True, null=True, max_length=50)
```

Comment:
Since we are syncing a minimal amount of fields for CW contacts, I think it's reasonable to do the same here. See comment at https://github.com/KerkhoffTechnologies/django-connectwise/pull/308#discussion_r542930266 . 

I think everything can be removed except: first_name, last_name, email_address and account.